pkg/rpc/client/client.go

```diff
@@ -145,47 +145,57 @@ func (c *Client) CalculateInputs(address string, asset util.Uint256, cost util.F
 }
 
 func (c *Client) performRequest(method string, p request.RawParams, v interface{}) error {
+	var r = request.Raw{
+		JSONRPC:   request.JSONRPCVersion,
+		Method:    method,
+		RawParams: p.Values,
+		ID:        1,
+	}
+
+	raw, err := c.makeHTTPRequest(&r)
+
+	if raw != nil && raw.Error != nil {
```

Comment:
Can ever `raw == nil` when `err != nil` too?

Reply:
We may have `raw != nil && err != nil` which is where I want `raw.Error` to have precedence over `err`. It's for cases like HTTP error on the lower layer with proper JSON `Error` inside in the contents.